app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    redirect_to user_post_path(id: @comment.post_id, user_id: @comment.author_id) if @comment.save
```

Comment:
- [ ] @V0pe great work on your create method, however, it would be nice if you had conditional rendering (eg if-else statement) for when the comment is created and when due to an error it's not created.

Reply:
@danmainah I added the if statement